bokeh/charts/builder/bar_builder.py

```diff
@@ -37,7 +37,40 @@
 
 def Bar(values, cat=None, stacked=False, xscale="categorical", yscale="linear",
         xgrid=False, ygrid=True, continuous_range=None, **kw):
+    """ Create a Bar chart using the `BarBuilder` to
```

Comment:
You are right. backticks are useless and :class: has no effect. Removing backticks..